config/settings.yml.example

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I think these options deserves some comments explaining their meaning, also I wonder why capital letters for Host and Port? I'd like to keep it consistent if possible